google/api_core/retry/retry_streaming_async.py

```diff
@@ -163,11 +163,15 @@ async def retry_target_stream(
 
                         # TODO: Remove this conditional once the minimum supported Python version is 3.11
                         if sys.version_info[:3] >= (3, 11, 0):
-                            await cast(AsyncGenerator["_Y", None], target_iterator).athrow(
-                                sys.exception())  # type: ignore
+                            await cast(
+                                AsyncGenerator["_Y", None], target_iterator
+                            ).athrow(
+                                sys.exception()  # type: ignore
+                            )
```

Comment:
it looks like we could just use sys.exc_info()[1] to be compatible with all versions?

`sys.exception()` looks cleaner, but I don't think it's worth the extra noise for the next few years

Reply:
Oh, that's a great idea. Let me try that.

Reply:
OK, thanks. That worked. Merging.